jaxrs-api/src/main/java/jakarta/ws/rs/container/Suspended.java

```diff
@@ -76,7 +76,9 @@
  *
  * @author Marek Potociar
  * @since 2.0
+ * @deprecated This class will be removed in a future version.  Better align with Jakarta CDI.
  */
+@Deprecated
```

Comment:
Should this be `@Deprecated(forRemoval = true)`?

Reply:
very true... thanks